FaceID/TestCode/camTest.py

```diff
@@ -0,0 +1,29 @@
+import cv2
+import numpy as np
+from PIL import Image
+import os
```

Comment:
Typically there are new lines after imports, can we do that here ( and for the other files in this MR)?